Cargo.toml

```diff
@@ -69,7 +79,64 @@ clokwerk = "0.3.5"
 doku = { version = "0.20.0", features = ["url-2"] }
 parking_lot = "0.12.1"
 reqwest-retry = "0.1.5"
+bcrypt = "0.13.0"
+chrono = { version = "0.4.22", features = ["serde"], default-features = false }
+serde_json = { version = "1.0.87", features = ["preserve_order"] }
+base64 = "0.13.1"
+uuid = { version = "1.2.1", features = ["serde", "v4"] }
+async-trait = "0.1.58"
+captcha = "0.0.9"
+anyhow = "1.0.66"
+diesel_ltree = "0.3.0"
+typed-builder = "0.10.0"
```

Comment:
Can't a lot of these dependencies use `workspace = true` also? Almost all of them are included in utils.

edit: okay I spose this seems good, since you're preferring to specify them higher up, then use them lower.

Reply:
I excluded deps which are only used in one of our crates. Seems like it would be more annoying than helpful to specify them in two places in that case.

Reply:
And only tools im using are cargo check/cargo tree.